cerbero/build/oven.py

```diff
@@ -191,6 +191,8 @@ def _cook_recipe(self, recipe, count, total, fridge=None, use_binaries=False, up
                 stepfunc = getattr(recipe, step)
                 if not stepfunc:
                     raise FatalError(_('Step %s not found') % step)
+                if step == BuildSteps.INSTALL[1]:
```

Comment:
The oven is dumb, it should only run step functions. We need to away to do this cleanly, either we a new step or adding it in the base class of the install step